website/source/guides/external/lxc.html.md

```diff
@@ -212,8 +144,6 @@ ID        Node ID   Task Group  Version  Desired  Status   Created    Modified
 4248c82e  81c22a0c  example     0        run      running  6m58s ago  6m47s ago
 ```
 
-## Next Steps
```

Comment:
The next section is not a step and still needs a nice heading. How about "More Config Options"